src/main/api-utilities/VerifierInterface.js

```diff
@@ -0,0 +1,21 @@
+export class VerifierInterface {
+    constructor(scenario, type) {
```

Comment:
In here, what is the type?

Reply:
the type of verificator, can be actor, action, weapon or scenario